android/src/com/frostwire/android/gui/activities/VPNStatusDetailActivity.java

```diff
@@ -55,20 +56,31 @@ protected void initComponents(Bundle savedInstanceState) {
             getActionBar().setDisplayHomeAsUpEnabled(true);
             getActionBar().setIcon(android.R.color.transparent);
         }
-
         final Intent intent = getIntent();
-        final boolean isProtectedConnection = intent.getAction() != null &&
+        isProtectedConnection = intent.getAction() != null &&
```

Comment:
you can revert this line, and declare the variable locally here, it's not used elsewhere, therefore no need to make it an instance's property.